packages/webpack/test/fixtures/main/policy-dependencies/policy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resources": {
+    "prototype-poisoning-package": {
+      "builtin": {
+        "path": true
+      },
+      "globals": {
+        "console.trace": true
```

Comment:
```suggestion
        "__webpack_require__": true
```

Reply:
nope, that's the point - it's explicitly not included in the policy and still works. That's inevitable (and we don't put `require` in policy as a global either)